tests/testthat/test-barpictograph.R

```diff
@@ -19,6 +19,7 @@ test_that("Warnings for invalid parameters",
 
 test_that("Custom image",
 {
+    skip_if_offline()
```

Comment:
These tests fail for the `custom.image` case because R does some additional checking and tries to call `GET` to download the image before passing it to rhtmlPictograph. For now, these tests can continue to run locally, and maybe we can restore it when we allow nix tests to have network access but I don't think these are that important.